translations/tiled_fr.ts

```diff
@@ -2595,7 +2596,7 @@ Voulez-vous le remplacer ?</translation>
     <message>
         <location line="-1223"/>
         <source>Ctrl+Shift+O</source>
-        <translation type="unfinished"></translation>
+        <translation>Ctrl+Màj+O</translation>
```

Comment:
Really no problem! I indeed noticed one earlier "Màj" and have fixed that one as part of the update. If you know about any other translated shortcuts, please do tell or submit a fix. :-)